live-comms/August-2021-Update.html

```diff
@@ -62,7 +62,7 @@
     </div>
     <nav aria-label="Primary navigation" class="usa-nav">
       <button class="usa-nav__close">
-        <img src="./close.svg" role="img" alt="Close" />
+        <img src="././img/close.svg" role="img" alt="Close" />
```

Comment:
routing is handled differently by github than in a typical static site, some extra magic was happening